core/cmd/helpers_test.go

```diff
@@ -6,3 +6,8 @@ import "github.com/smartcontractkit/chainlink/core/logger"
 func (cli *Client) CheckRemoteBuildCompatibility(lggr logger.Logger, onlyWarn bool, cliVersion, cliSha string) error {
 	return cli.checkRemoteBuildCompatibility(lggr, onlyWarn, cliVersion, cliSha)
 }
+
+// ConfigDumpHelper exposes configDump for testing.
+func (cli *Client) ConfigDumpHelper() (string, error) {
```

Comment:
Again open to suggestions on naming. Better to call the original one *Helper ...?

Reply:
I think this is fine, but it might also be fitting to align unexported and helper as a Str/String variant :shrug: 
```go
func (cli *Client) ConfigDumpStr() (string, error) {
	return cli.configDumpStr()
```